package.json

```diff
@@ -185,6 +187,7 @@
     "react-router-dom": "^4.2.2",
     "react-select": "^1.0.0",
     "react-simple-file-input": "^2.0.0",
+    "react-switch": "^5.0.0",
```

Comment:
Hi @andrewnguyen22 ! Do we need `react-switch`? I don't see it is being used on the code. Same question goes for `@babel/core`

Reply:
See https://github.com/poanetwork/nifty-wallet/pull/303#issuecomment-507727204